packages/modules/src/modules/banners/designableBanner/components/DesignableBannerVisual.tsx

```diff
@@ -63,9 +63,11 @@ const getStyles = (isHeaderImage = false) => {
 
                 img {
                     height: 100%;
-                    width: 100%;
                     object-fit: contain;
                     display: block;
+                    margin-left: -10px;
+                    margin-right: -10px;
+                    width: calc(100% + 20px);
```

Comment:
is there a way to make it full width from the parent element?

Reply:
just for my own understanding could you expand why we might want to make it full width from the parent element? The change here makes sense to me as it's sitting on the image component and it clear what the width will be full width + 20px if it is a header image

Reply:
I just found it confusing that the width is more that 100%, but it makes sense 👍